### PR TITLE
Ed 2385 add triage form view

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-2385)

wizard

image

### CODE REVIEW

File: core/templates/core/widgets/radio.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% with id=widget.attrs.id %}<ul{% if id %} id="{{ id }}"{% endif %}{% if widget.attrs.class %} class="{{ widget.attrs.class }}"{% endif %}>{% for group, options, index in widget.optgroups %}{% if group %}
```
Comment (konradko) [inline core/templates/core/widgets/radio.html]: very unreadable, please pretty print

Comment (richtier) [reply core/templates/core/widgets/radio.html]: this is copied and pasted from django's codebase with a few change. keeping it similar structure to theirs makes diffing with their file easier

Comment (konradko) [reply core/templates/core/widgets/radio.html]: why would you keep diffing?

Comment (richtier) [reply core/templates/core/widgets/radio.html]: someone other than myself may want to understand the differences between this file and Django's file - e.g., during an upgrade to Django 1.2.0 when Django's template may change.

**Pull request merged**